### ISSUE TITLE
Subcategory possibility for KB module

### ISSUE THREAD

**Issue by Mausino** [salesagility/SuiteCRM#725]
Hi, 

i think that for bigger companies, which have many departments + many teams we need Main + Subcategories and for this reason is nessary  to create some subcategory module for KB like we have now for the Products categories module.

What are you thinking about it ?

Miky

**Comment by ghermans** [comment]
+1

**Comment by samus-aran** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Subcategory possibility for KB module](https://trello.com/c/bFvv3VGH/131-subcategory-possibility-for-kb-module)

**Comment by samus-aran** [comment]
We have now set up a new home for suggestions at Trello.  All github issues that were labeled 'suggestion' have been moved and will be closed.  Certain ones will be progressed within the new Suggestion Box and may be re-opened.

Announcement of moving Suggestions:
https://suitecrm.com/forum/suggestion-box/13691-moving-suggestions-from-github-to

New SuiteCRM Suggestion Box
https://trello.com/b/Ht7LbMqw/suitecrm-suggestion-box
